uint16 port
string return_address
string source_topic
string destination_topic
uint32 queue_size
float32 period
